packages/rich-text/src/store/selectors.js

```diff
@@ -37,9 +37,15 @@ export function getFormatType( state, name ) {
  * @return {?Object} Format type.
  */
 export function getFormatTypeForBareElement( state, bareElementTagName ) {
-	return getFormatTypes( state ).find( ( { className, tagName } ) => {
-		return className === null && bareElementTagName === tagName;
-	} );
+	const formatTypes = getFormatTypes( state );
+	return (
+		formatTypes.find( ( { className, tagName } ) => {
+			return className === null && bareElementTagName === tagName;
+		} ) ||
+		formatTypes.find( ( { className, tagName } ) => {
+			return className === null && '*' === tagName;
```

Comment:
This line ensures that `core/unknown` is detected.
Therefore, it appears that format types that don't have `className` cannot be newly registered, even if the `tagName` has not yet been registered.

Reply:
Fixed by #46798